osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/infra/messaging/DlmsLogItemRequestObjectMessageCreatorTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package org.opensmartgridplatform.adapter.protocol.dlms.infra.messaging;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import javax.jms.JMSException;
+import javax.jms.Message;
+import javax.jms.ObjectMessage;
+import javax.jms.Session;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class DlmsLogItemRequestObjectMessageCreatorTest {
+  private Session session;
+
+  private DlmsLogItemRequestMessage dlmsLogItemRequestMessage;
+  private DlmsLogItemRequestObjectMessageCreator dlmsLogItemRequestObjectMessageCreator;
+
+  @BeforeEach
+  public void setUp() {
+    this.dlmsLogItemRequestMessage = this.getDlmsLogItemRequestMessage();
+    this.dlmsLogItemRequestObjectMessageCreator =
+        new DlmsLogItemRequestObjectMessageCreator(this.dlmsLogItemRequestMessage);
+    this.session = mock(Session.class);
+  }
+
+  @Test
+  void createObjectMessage() throws JMSException {
+    final ObjectMessage expectedObjectMessage = mock(ObjectMessage.class);
+    when(this.session.createObjectMessage()).thenReturn(expectedObjectMessage);
+
+    final Message actualMessage =
+        this.dlmsLogItemRequestObjectMessageCreator.getObjectMessage(this.session);
+
+    assertThat(actualMessage).isSameAs(expectedObjectMessage);
```

Comment:
JMSType setting is not verified (see comments in other message creator test classes for more detailed guidance).